doc/en/components/grids/_shared/row-editing.md

```diff
@@ -92,17 +92,17 @@ public unitsInStockCellTemplate = (ctx: IgcCellTemplateContext) => {
 function unitsInStockCellTemplate(ctx: IgrCellTemplateContext) {
     return (
         <>
-            <input name="units" value={ctx.dataContext.cell.value} style={{color: "black"}} />;
+            <input name="units" value={ctx.cell.value} style={{color: "black"}} />;
         </>
     );
 }
 
 <{ComponentSelector} primaryKey="ProductID" width="100%" height="500px" rowEditable="true"
-    bodyTemplate={unitsInStockCellTemplate}>
+    >
```

Comment:
There are boolean properties that are set as strings

Reply:
fixed